website/_sass/color_schemes/_variables.scss

```diff
@@ -1 +1,5 @@
+// Overrides for Just the Docs, see
+// https://github.com/just-the-docs/just-the-docs/blob/main/_sass/support/_variables.scss.
+
+// Grid system
```

Comment:
Should we also add a verb here? At least we should add a `.`.

Reply:
It is rather a categorization, I believe. So I'm happy to leave this. But will add `.`.

Reply:
Nice.